app/templates/components/filterable-resource.hbs

```diff
@@ -1,13 +1,18 @@
-{{paper-input label="Search" value=filter onChange=(action 'autoComplete' filter) icon="search"}}
-
-{{#if hasBlock}}
-  {{yield}}
-{{else}}
-  <ul>
-    {{#each filteredList as |item|}}
-      <li>
-        {{link-to item.name modelName item}}
```

Comment:
Removing this breaks the linkage to the appropriate model. When someone selects a user they should go to `user.scorecards`. When someone selects a course, it should go to `course.scorecards`.